src/db/spanner/models.rs

```diff
@@ -985,7 +1028,18 @@ impl SpannerDb {
                AND collection_id = @collection_id
                AND expiry > CURRENT_TIMESTAMP()";
         let limit = params.params.limit.map(i64::from).unwrap_or(-1);
-        let offset = params.params.offset.unwrap_or(0) as i64;
+        let offset = params
+            .params
+            .offset
+            .clone()
+            .unwrap_or(Offset {
+                offset: 0,
+                timestamp: None,
+            })
+            .offset;
+        let sort = params.params.sort.clone();
+        let newer_eq = params.params.newer_eq;
```

Comment:
This is missing integration of `newer/older_eq` into the `bsos_query_sync's` query:

https://github.com/mozilla-services/server-syncstorage/blob/576490a/syncstorage/storage/sql/queries_spanner.py#L96

Reply:
🤦‍♂️